src/components/filter_group/filter_select_item.tsx

```diff
@@ -128,6 +144,23 @@ export class EuiFilterSelectItemClass extends Component<
         </EuiFlexGroup>
       </button>
     );
+
+    return hasToolTip ? (
+      // This extra wrapper is needed to ensure that the tooltip has a correct context
+      // for positioning while also ensuring to wrap the interactive option
+      <span style={style}>
+        <EuiToolTip
+          display="block"
+          content={toolTipContent}
```

Comment:
This is a spicy potato suggestion but how do you feel about defaulting the tooltip alignment to the left instead of bottom (so it doesn't occlude other options?)
```suggestion
          position="left"
          content={toolTipContent}
```

Reply:
I agree that makes the experience better! It's not the default position of the tooltip and it's not currently defined but it has for sure value. 👍